target/linux/ath79/dts/ar9344_mikrotik_routerboard-2011.dts

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "mikrotik,routerboard-2011", "qca,ar9344";
+	model = "MikroTik RouterBOARD 2011";
+};
+
+&spi {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <40000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "RouterBoot";
+				reg = <0x0 0x10000>;
+				read-only;
+				compatible = "mikrotik,routerboot-partitions";
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partition@0 {
+					label = "bootloader1";
+					reg = <0x0 0x0>;
+					read-only;
+				};
+
+				hard_config: hard_config {
+					read-only;
+				};
+
+				bios {
+					size = <0x1000>;
+					read-only;
+				};
+
+				soft_config {
+				};
+			};
+		};
+	};
+};
+
+&nand {
+	status = "okay";
+
+	nand-ecc-mode = "soft";
+	qca,nand-swap-dma;
+	qca,nand-scan-fixup;
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			label = "booter";
+			reg = <0x0000000 0x0040000>;
+			read-only;
+		};
+
+		partition@40000 {
+			label = "kernel";
+			reg = <0x0040000 0x03c0000>;
+		};
+
+		partition@400000 {
+			label = "ubi";
+			reg = <0x0400000 0x7c00000>;
+		};
+	};
+};
+
+&uart {
+	status = "okay";
+};
+
+&mdio0 {
+	status = "okay";
+
+	phy-mask = <0>;
+
+	phy0: ethernet-phy@0 {
+		reg = <0>;
+		phy-mode = "rgmii-id";
+
+		qca,ar8327-initvals = <
+			0x04 0x07800000 /* PORT0 PAD MODE CTRL */
+			0x10 0x81000080 /* POWER_ON_STRAP */
+			0x50 0xc731c731 /* LED_CTRL0 */
+			0x54 0x00000000 /* LED_CTRL1 */
+			0x58 0x00000000 /* LED_CTRL2 */
+			0x5c 0x0030c300 /* LED_CTRL3 */
+			0x7c 0x0000007e /* PORT0 STATUS */
+			0x0c 0x00000080 /* PORT6 PAD Mode */
+			0xe0 0xc70167d0 /* SGMII_CTRL */
+			/*0x94 0x0000007e PORT6_STATUS */
+		>;
+	};
+};
+
+&eth0 {
+	status = "okay";
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	/* default for ar934x, except for 1000M */
+	pll-data = <0x6f000000 0x00000101 0x00001616>;
+
+	phy-mode = "rgmii-id";
+	phy-handle = <&phy0>;
+
+	/*gmac-config {
+		device = <&gmac>;
+		rgmii-enabled = <1>;
```

Comment:
```rgmii-enabled``` property in ```gmac-config``` is not valid for AR934x, it can be dropped.  
ref: https://github.com/openwrt/openwrt/blob/2160a9d597b7bb7b14a65785117ff5891db518ef/target/linux/ath79/files/drivers/net/ethernet/atheros/ag71xx/ag71xx_gmac.c#L50-L71